implementations/rust/ockam/ockam_command/src/secure_channel/listener/create.rs

```diff
@@ -37,7 +37,8 @@ pub struct CreateCommand {
     #[arg(value_name = "VAULT", long, requires = "identity")]
     vault: Option<String>,
 
-    #[arg(value_name = "IDENTITY", long)]
+    //Name of the Identity that the secure-channel listener will use
```

Comment:
This should be `/// Name ...` (note the extra `/` and the space)

Reply:
Ohh ,forgot to put the third `/`. Fixed it.